src/math/arithmetic.rs

```diff
@@ -0,0 +1,128 @@
+//! Arithmetics module implements Numpy routines listed at
+//! https://docs.scipy.org/doc/numpy/reference/routines.math.html#arithmetic-operations
+//!
+//!
+
+use ndarray::*;
+use num_traits;
+use std;
+
+/// add
+/// takes two ndarrays and adds them
+#[allow(unused)]
+pub fn add<A, D>(arr1: &Array<A, D>, arr2: &Array<A, D>) -> Array<A, D>
```

Comment:
I don't think free functions are the right way to expose operations like this. They really should be methods, or even an overloading of the `+` operator.